medusa/providers/torrent/html/bjshare.py

```diff
@@ -0,0 +1,250 @@
+                log.debug("Data returned from provider does not contain any torrents")
+                return {'has_next_page': has_next_page, 'items': []}
+
+            # "", "", "Name /Year", "Files", "Time", "Size", "Snatches", "Seeders", "Leechers"
+            labels = [self._process_column_header(label) for label in torrent_rows[0]("td")]
+
+            # Skip column headers
+            for result in torrent_rows[1:]:
+                cells = result("td")
+                if len(cells) < len(labels):
+                    continue
+                try:
+                    title = cells[labels.index("Nome/Ano")].find("a", dir="ltr").get_text(strip=True)
+                    # get international title if available
+                    title = re.sub(".* \[(.*?)\](.*)", r"\1\2", title)
+
+                    for serie in self.absolute_numbering:
+                        if serie in title:
+                            # remove season from title when its in absolute format
+                            title = re.sub("S\d{2}(E\d{2,4})", r"\1", title)
+                            break
+
+                    download_url = urljoin(self.url, cells[labels.index("Nome/Ano")].find("a", title="Baixar")["href"])
+                    if not all([title, download_url]):
+                        continue
+
+                    seeders = try_int(cells[labels.index("Seeders")].get_text(strip=True))
+                    leechers = try_int(cells[labels.index("Leechers")].get_text(strip=True))
+
+                    # Filter unseeded torrent
+                    if seeders < self.minseed or leechers < self.minleech:
+                        if mode != "RSS":
+                            log.debug("Discarding torrent because it doesn't meet the"
+                                      " minimum seeders or leechers: {0} (S:{1} L:{2})".format
+                                      (title, seeders, leechers))
+                        continue
+
+                    torrent_details = cells[labels.index("Nome/Ano")].find("div", attrs={
+                        "class": "torrent_info"}).get_text(strip=True).replace('[', ' ').replace(']', ' ')\
+                        .replace('/', ' ')
+
+                    for key, value in self.quality.iteritems():
+                        torrent_details = torrent_details.replace(key, value)
+
+                    torrent_size = cells[labels.index("Tamanho")].get_text(strip=True)
+                    size = convert_size(torrent_size, units=self.units) or -1
+
+                    torrent_name = "{0} {1}".format(title, torrent_details.strip()).strip()
+                    torrent_name = re.sub('\s+', ' ', torrent_name)
+
+                    items.append({
+                        'title': torrent_name,
+                        'link': download_url,
+                        'size': size,
+                        'seeders': seeders,
+                        'leechers': leechers,
+                        'hash': ''
```

Comment:
Remove `hash`, add `pubdate`.

Reply:
This tracker does not include pubdate in the torrent listing page...
hash removed